docs/cli.md

```diff
@@ -165,6 +171,23 @@ And then require it using the `--require` option:
 
 Note that since the first `--require babel.register.js` overrides the default require glob, we redeclare it with `--require 'features/**/*.js'`.
 
+Similarly, with TypeScript, you might want to provide configuration that prevents the compiled JS being written out to files, and pass some compiler options:
+
+```js
+require('ts-node').register({
+  transpileOnly: true,
+  compilerOptions: {
+    // your compiler options here
+  },
+});
+```
```

Comment:
I don't see how this adds anything that isn't shown by the babel example. Would rather keep just one (fine updating that to typescript since plan to move cucumber that way anyway)

Reply:
Consolidated to just the TS example